package.json

```diff
@@ -59,7 +59,7 @@
     "proxyquire": "^1.6.0",
     "proxyquire-universal": "^1.0.8",
     "proxyquireify": "^3.0.0",
-    "sinon": "^1.15.4"
+    "sinon": "1.16.1"
```

Comment:
This is to fix tests that were failing on Travis but not locally, due to Sinon differences